### ISSUE TITLE
Auto-rig & IK not working

### ISSUE THREAD

**Issue by SaraMac87** [RxLaboratory/DuAEF_Duik#680]
Seems as though magic doesn't happen in the newest updated version of duik.
Was creating an arm rig and hit "Auto-rig & IK". It never rigged :(

**Comment by Nico-Duduf** [comment]
I'm going to need more details to understand what's going on....
Exact version of Duik, of After Effects, OS, context, maybe the aep, etc...

**Comment by SaraMac87** [comment]
OS Catalina 10.15.3
After Effects 2020
Duik Bassel.2 v16.2.10 (updated this morning)

I was attempting a rig exactly like the one on [this video](https://youtu.be/N53TEH--OuA?t=1899), which I was able to achieve yesterday, before updating Duik Bassel. 
After this morning's attempt, it doesn't move any further than "magic is happening".
I have restarted AE and my laptop, to no avail, and have also tried to install the older version of Duik Bassel.2 but maybe I'm doing it wrong.

I don't have a lot of tech knowledge, if you'd be so kind as to dumb it down, I'd appreciate for life. Thank you for you patience and help in advance and please let me know if you still need the AE.

Thanks!
Sara

**Comment by Nico-Duduf** [comment]
Could you just send me the aep as it is before running the auto-rig? I'll try on my side first to see if I can reproduce the issue.
You can also go to the After Effects preferences/scripting, enable javascript debugging there, and post a screenshot of the error/window shown when you try again the auto-rig, this may help find the bug!

**Comment by malconfrcs** [comment]
Its happening to me.  "Auto-rig & IK"doesnt rig.The composition MAGIC IS HAPPENING with the magic wand appears and nothing else happens.

**Comment by malconfrcs** [comment]
> I need a failing .aep to test

Solvedt. It was my silly mistake. I was selecting wrong layers for the rig The controlling layers instead of the shapes. Thanks for your time.
